autoload/tagbar.vim

```diff
@@ -947,6 +947,10 @@ function! s:InitWindow(autoclose) abort
     setlocal nomodifiable
     setlocal textwidth=0
 
+    if g:tagbar_scroll_offset > 0
```

Comment:
Can we call this `g:tagbar_scrolloff` to match the vim setting name that actually implements it?

Reply:
done